controllers/admin/AdminOrdersController.php

```diff
@@ -2125,6 +2125,8 @@ public function ajaxProcessAddProductOnOrder()
         // Total method
         $total_method = Cart::BOTH_WITHOUT_SHIPPING;
 
+        Db::getInstance()->beginTransaction();
```

Comment:
This kind of translations must be done only if innodb is used

Reply:
Good point! I'll fix that!
> If the underlying driver does not support transactions, a PDOException will be thrown (regardless of your error handling settings: this is always a serious error condition).

Reference: https://www.php.net/manual/en/pdo.transactions.php

Reply:
@PierreRambaud: I've checked PHP documentation and it says it's actually **safe** to use transactions even with MyISAM. This code **won't** trigger an exception. I've tested it with PrestaShop `develop` branch running on MySQL 5.7.35 with all tables using MyISAM.